.github/workflows/stale.yml

```diff
@@ -0,0 +1,32 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '30 4 * * *'
+permissions:
+  issues: write
+  pull-requests: write
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v8
+        with:
+          # Number of days of inactivity before a stale Issue or Pull Request is closed.
+          # Set to -1 to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+          days-before-close: 60
+          # Number of days of inactivity before an Issue or Pull Request becomes stale
+          days-before-stale: 90
```

Comment:
Are these numbers in line with the stale and close messages?

Reply:
they are now, thanks.